Convex_hull_3/examples/Convex_hull_3/CMakeLists.txt

```diff
@@ -42,6 +42,8 @@ endif()
 # Creating entries for all C++ files with "main" routine
 # ##########################################################
 
+create_single_source_cgal_program( "is_strongly_convex_3.cpp" )
```

Comment:
you did not commit the test

Reply:
Let me remove it again.  In fact it shows that our testsuite globally does not address these constness issues.